Fix a panic when creating HVS app or secret (#1154)
hashicorp/terraform-provider-hcp: .changelog/1154.txt

```release-note:bug

> creating a hvs app or kv secret should not panic

Reviewer: _nit_: I'd rather just repeat the PR title here

```suggestion
Fix a panic when creating HVS app or secret
```

Author: done
